Dockerfile

```diff
@@ -45,11 +45,21 @@ ENV     MEILI_HTTP_ADDR 0.0.0.0:7700
 ENV     MEILI_SERVER_PROVIDER docker
 
 RUN     apk update --quiet \
-        && apk add -q --no-cache libgcc tini curl
+        && apk add -q --no-cache libgcc tini
 
-COPY    --from=compiler /meilisearch/target/release/meilisearch .
+# add meilisearch to the `/bin` so you can run it from anywhere and it's easy
+# to find.
+COPY    --from=compiler /meilisearch/target/release/meilisearch /bin/meilisearch
+
+# We want user to mount this directory as a volume
+RUN     mkdir /data
```

Comment:
So I'm not really sure what's going on here, we make a directory first but then also tag it as our working directory.
If we're marking this as our WD then we don't need to make the directory first, it's just an implicit step iirc.

Reply:
Oh cool, I didn't know that!
Thanks :+1: